Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM clojure:lein-2.5.1
+FROM quay.io/democracyworks/didor
+MAINTAINER Democracy Works, Inc. <dev@democracy.works>
 
-RUN mkdir -p /usr/src/usps-processor
-WORKDIR /usr/src/usps-processor
+ADD project.clj /usps-processor/
+WORKDIR /usps-processor
 
-COPY profiles.clj /usr/src/usps-processor/
-COPY project.clj /usr/src/usps-processor/
+ADD lein_deps /usps-processor/
 
-RUN lein deps
+RUN bash /usps-processor/lein_deps
 
-COPY . /usr/src/usps-processor
+ADD ./ /usps-processor/
+
+RUN rm /usps-processor/lein_deps
 
 RUN lein test
-RUN mv "$(lein uberjar | sed -n 's/^Created \(.*standalone\.jar\)/\1/p')" usps-processor-standalone.jar
 
-CMD ["java", "-jar", "usps-processor-standalone.jar"]
+EXPOSE 8080
```

Comment:
We should build the war file in here w/ `RUN lein immutant war --name usps-processor --destination target`.